common-test/src/main/java/io/strimzi/test/k8s/BaseKubeClient.java

```diff
@@ -297,6 +312,14 @@ public String toString() {
         return asList(Exec.exec(namespacedCommand("get", resourceType, "-o", "jsonpath={range .items[*]}{.metadata.name} ")).out().trim().split(" +"));
     }
 
+    public List<String> listByLabel(String resourceType, String labelName) {
```

Comment:
with this name it seems that you can list by any "label" but it lists just for the label named "name"; maybe `listByNameLabel` or something like that ? and then having the second parameter just `name` ? But it's not a change to do ... just my impression :-)

Reply:
@ppatierno Thanks for pointing me here. Actually, it's unused method. Removing it.

Reply:
@asorokhtey I knew that :P (I'm cheating ...)